package.json

```diff
@@ -3,54 +3,25 @@
   "version": "1.0.0",
   "author": "Optimism PBC",
   "license": "MIT",
-  "workspaces": {
-    "packages": [
-      "packages/*",
-      "batch-submitter",
-      "bss-core",
-      "gas-oracle",
-      "integration-tests",
-      "l2geth-exporter",
-      "l2geth",
-      "ops/docker/rpc-proxy",
-      "ops/docker/hardhat",
-      "ops/docker/go-builder",
-      "ops/docker/js-builder",
-      "ops/docker/ci-builder",
-      "ops/docker/foundry",
-      "endpoint-monitor"
-    ],
-    "nohoist": [
-      "**/typechain/*",
-      "**/@typechain/*",
-      "@eth-optimism/contracts-bedrock/ds-test",
-      "@eth-optimism/contracts-bedrock/forge-std",
-      "@eth-optimism/contracts-bedrock/@rari-capital/solmate",
-      "**/@openzeppelin/*",
-      "@eth-optimism/contracts-periphery/ds-test",
-      "@eth-optimism/contracts-periphery/forge-std",
-      "@eth-optimism/contracts-periphery/@rari-capital/solmate",
-      "forta-agent"
-    ]
-  },
   "private": true,
+  "engines": {
+    "pnpm": ">=7"
```

Comment:
would be a good idea to also add

"node": ">16",

but that is out of scope for this pr

Reply:
this will make it so if user uses npm or yarn, everything will work fine but they will get a warning in console that pnpm is the correct engine to use